[2023-02-21 21:06:32,991] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.scraping manual__2023-02-21T21:06:29.407876+00:00 [queued]>
[2023-02-21 21:06:33,004] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.scraping manual__2023-02-21T21:06:29.407876+00:00 [queued]>
[2023-02-21 21:06:33,005] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-02-21 21:06:33,006] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-02-21 21:06:33,007] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-02-21 21:06:33,022] {taskinstance.py:1259} INFO - Executing <Task(_PythonDecoratedOperator): scraping> on 2023-02-21 21:06:29.407876+00:00
[2023-02-21 21:06:33,029] {standard_task_runner.py:52} INFO - Started process 353 to run task
[2023-02-21 21:06:33,036] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'scraping', 'manual__2023-02-21T21:06:29.407876+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/reddit_ingestion_dag.py', '--cfg-path', '/tmp/tmpssgfw3zh', '--error-file', '/tmp/tmpf47xaheq']
[2023-02-21 21:06:33,037] {standard_task_runner.py:77} INFO - Job 10: Subtask scraping
[2023-02-21 21:06:33,117] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.scraping manual__2023-02-21T21:06:29.407876+00:00 [running]> on host b46f753f04a9
[2023-02-21 21:06:33,372] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=scraping
AIRFLOW_CTX_EXECUTION_DATE=2023-02-21T21:06:29.407876+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-21T21:06:29.407876+00:00
[2023-02-21 21:06:33,377] {logger.py:11} INFO - ====== WebDriver manager ======
[2023-02-21 21:06:33,606] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome 110.0.5481
[2023-02-21 21:06:33,924] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/110.0.5481/chromedriver] found in cache
[2023-02-21 21:06:51,891] {sessions.py:161} WARNING - Retrying due to 503 status: GET https://oauth.reddit.com/comments/108zqmx/
[2023-02-21 21:06:53,207] {sessions.py:161} WARNING - Retrying due to 503 status: GET https://oauth.reddit.com/comments/108zqmx/
[2023-02-21 21:06:55,575] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 134, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/reddit_ingestion_dag.py", line 110, in scrape_test
    scrape_keyword_from_reddit(['weclouddata'])
  File "/opt/airflow/dags/scraper.py", line 136, in scrape_keyword_from_reddit
    result = search_keywords_in_reddit(cmp)
  File "/opt/airflow/dags/scraper.py", line 73, in search_keywords_in_reddit
    print(idx, " -- ", submission.title, " -- ", submission.subreddit)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/base.py", line 34, in __getattr__
    self._fetch()
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/submission.py", line 634, in _fetch
    data = self._fetch_data()
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/submission.py", line 631, in _fetch_data
    return self._reddit.request(method="GET", params=params, path=path)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/util/deprecate_args.py", line 43, in wrapped
    return func(**dict(zip(_old_args, args)), **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/reddit.py", line 947, in request
    path=path,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 337, in request
    url=url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 263, in _request_with_retries
    url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 170, in _do_retry
    retry_strategy_state=retry_strategy_state.consume_available_retry(),  # noqa: E501
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 263, in _request_with_retries
    url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 170, in _do_retry
    retry_strategy_state=retry_strategy_state.consume_available_retry(),  # noqa: E501
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 266, in _request_with_retries
    raise self.STATUS_EXCEPTIONS[response.status_code](response)
prawcore.exceptions.ServerError: received 503 HTTP response
[2023-02-21 21:06:55,645] {taskinstance.py:1277} INFO - Marking task as UP_FOR_RETRY. dag_id=tutorial, task_id=scraping, execution_date=20230221T210629, start_date=20230221T210632, end_date=20230221T210655
[2023-02-21 21:06:55,697] {standard_task_runner.py:92} ERROR - Failed to execute job 10 for task scraping
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 134, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/reddit_ingestion_dag.py", line 110, in scrape_test
    scrape_keyword_from_reddit(['weclouddata'])
  File "/opt/airflow/dags/scraper.py", line 136, in scrape_keyword_from_reddit
    result = search_keywords_in_reddit(cmp)
  File "/opt/airflow/dags/scraper.py", line 73, in search_keywords_in_reddit
    print(idx, " -- ", submission.title, " -- ", submission.subreddit)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/base.py", line 34, in __getattr__
    self._fetch()
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/submission.py", line 634, in _fetch
    data = self._fetch_data()
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/models/reddit/submission.py", line 631, in _fetch_data
    return self._reddit.request(method="GET", params=params, path=path)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/util/deprecate_args.py", line 43, in wrapped
    return func(**dict(zip(_old_args, args)), **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/praw/reddit.py", line 947, in request
    path=path,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 337, in request
    url=url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 263, in _request_with_retries
    url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 170, in _do_retry
    retry_strategy_state=retry_strategy_state.consume_available_retry(),  # noqa: E501
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 263, in _request_with_retries
    url,
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 170, in _do_retry
    retry_strategy_state=retry_strategy_state.consume_available_retry(),  # noqa: E501
  File "/home/airflow/.local/lib/python3.7/site-packages/prawcore/sessions.py", line 266, in _request_with_retries
    raise self.STATUS_EXCEPTIONS[response.status_code](response)
prawcore.exceptions.ServerError: received 503 HTTP response
[2023-02-21 21:06:55,754] {local_task_job.py:154} INFO - Task exited with return code 1
[2023-02-21 21:06:56,001] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
